lib/assets/connection_cell/main.js

```diff
@@ -562,9 +562,11 @@ export function init(ctx, info) {
     methods: {
       handleFieldChange(event) {
         const { name, value } = event.target;
+        const isCheckbox = event.target.type === "checkbox";
+        const checked = event.target.checked;
 
         if (name) {
```

Comment:
Doesn't this work?

```suggestion
        const field = event.target.name;

        if (field) {
          const value = this.fields[field];
```

Reply:
Yes! I think I found the problem! The `$emit` was not emiting the modelValue